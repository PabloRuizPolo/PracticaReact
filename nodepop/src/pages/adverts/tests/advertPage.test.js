import { render } from "@testing-library/react";

import AdvertPage from "../advertPage";
import { Provider } from "react-redux";

describe("advertPage", () => {
  const state = {
    adds: {
      loaded: false,
      data: [],
    },
  };
  const store = {
    getState: () => state,
    dispatch: () => {},
    subscribe: () => {},
  };
  const renderComponents = () =>
    render(
      <Provider store={store}>
        <AdvertPage />
      </Provider>
    );

  test("snapshot", () => {
    const { container } = renderComponents();
    expect(container).toMatchSnapshot();
  });
});
