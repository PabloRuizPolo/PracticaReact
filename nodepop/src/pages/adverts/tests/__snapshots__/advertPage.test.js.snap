// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`advertPage snapshot 1`] = `
<div>
  <div>
    <div>
      <p>
        Cargando detalles del anuncio
      </p>
    </div>
  </div>
</div>
`;
